notebooks/scenarios/bigquery/001-scale-delete-worker-pools.ipynb

```diff
@@ -67,13 +81,30 @@
     "    port=\"8080\",\n",
     "    n_consumers=num_workers,  # How many workers to be spawned\n",
     "    create_producer=True,  # Can produce more workers\n",
+    "    reset=True,\n",
```

Comment:
MAYBE: we could remove this reset=True, as currently we reset db and load checkpoints?